README.md

```diff
@@ -1,24 +1,26 @@
-# README
+# Unnamed-Rails-Project
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+- Centralized token-like credits issuance platform for marketing purpose. Give tokens to your audience. Pay them for sharing, comments and other kind of work for you. Use token to build stronger community. Let yout people decide about your work and your content.
 
-Things you may want to cover:
+[![Build Status](http://img.shields.io/travis/badges/badgerbadgerbadger.svg?style=flat-square)](https://travis-ci.org/badges/badgerbadgerbadger) [![License](http://img.shields.io/:license-mit-blue.svg?style=flat-square)](http://badges.mit-license.org)
```

Comment:
Travis build status should be a part of travis PR. Please remove it from that one